longevity_large_partition_test.py

```diff
@@ -9,9 +9,10 @@ def test_large_partition_longevity(self):
         self._pre_create_schema()
         self.test_custom_time()
 
-    def _pre_create_schema(self, in_memory=False):
+    def _pre_create_schema(self, in_memory=False):  # pylint: disable=arguments-differ
```

Comment:
`*args, **kwargs`

Reply:
pylint didn't liked  *arg, **kwargs, i.e. the signature is still different